website/src/components/Users.tsx

```diff
@@ -31,11 +31,6 @@ const USERS: UserDetails[] = [
     name: "Metamask",
     link: "https://github.com/MetaMask/snaps-directory",
   },
-  {
-    logo: "uniswap.png",
```

Comment:
Could you also remove the `website/static/img/users/uniswap.png`? It's no longer used.

Reply:
done 🤦 😅